e2e/cta.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from '@playwright/test'
+import { test } from './base-test'
+
+test.use({
+  storageState: 'e2e/auth.json',
+})
+
+test('Call to action has rendered in dashboard page', async ({ page }) => {
+  await page.goto('/dashboard')
+
+  await expect(page.locator('text=Biaya kelas').first()).toBeVisible()
+  await expect(
+    page.locator('text=Terbatas untuk 30 orang peserta').first()
+  ).toBeVisible()
+
+  const purchaseButton = page.locator('a:has-text("Daftarkan diri")').first()
```

Comment:
Please use `getByRole` query from `playwright-testing-library`.
You can use [Testing Playground](https://testing-playground.com)'s browser extensions to get the recommended query easily.

You can refer to [settings.spec.ts](https://github.com/zainfathoni/kelas.rumahberbagi.com/pull/60/files?authenticity_token=l%2BWR4je6AIxYGVwXMLA5JSn8uzfsAljBhdUZCzLY9pekTkk1%2B7ib7SFNAMYBKhT0bc5MN3lkEAv4gJk7Hrenbw%3D%3D&file-filters%5B%5D=.ts&hide-deleted-files=true#diff-c1dd7f901a5034e85bf5308f50eefd5c35ec9c44c146eef8465355580a245e5bR18-R20) as an example.

Reply:
But, `toBeVisible()` must be used with Locator object, because I'm getting error `Error: toBeVisible can be only used with Locator object` when using `getByRole`

```ts
const heading = await getByRole('heading', {
    name: /biaya kelas/i,
})
await expect(heading).toBeVisible()
```

Reply:
Because I can't using `toBeVisible()`, can I write it like this?

```ts
const heading = await getByRole('heading', {
  name: 'Biaya kelas',
})
await test.expect(heading).not.toBeUndefined()
```

Reply:
I encountered the same issue while I was trying to do visibility assertions.

Unfortunately, we need to keep using Playwright's `Locator` feature for visibility assertions until we get a better solution. 😅
Hence I'm only asking you to refactor the parts where you're trying to interact with the elements, not assert the elements' visibility. 😁

Let's work on that problem in a separate GitHub issue. 😉

Reply:
I'm sorry wkwk, I tried that because you marked lines 13-14, that's why I thought to change that too 🤣

Reply:
Oops, my bad. Sorry about that. 😅🙏

Reply:
I'm not change line 16 because `toHaveAttribute()` can be only used with Locator object too